.eslintrc.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
   overrides: [
     {
       files: ['*.ts'],
+      excludedFiles: ['packages/ember/version.d.ts', 'vite.config.ts'],
```

Comment:
type-aware lints are included in here, but these file have no associated tsconfig.json, and thus cannot use type-aware lints.